extensions/mssql/src/objectManagement/ui/serverPropertiesDialog.ts

```diff
@@ -70,7 +69,7 @@ export class ServerPropertiesDialog extends ObjectManagementDialogBase<Server, S
 	}
 
 	protected override onFormFieldChange(): void {
```

Comment:
The base `onFormFieldChange` in scriptableDialogBase already has the same logic, so you can probably just remove this override method.